Formula/ship.rb

```diff
@@ -0,0 +1,30 @@
+class Ship < Formula
+  desc "Reducing the Overhead of Maintaining 3rd-party Applications in Kubernetes"
+  homepage "https://ship.replicated.com"
+  url "https://github.com/replicatedhq/ship/archive/v0.25.2.tar.gz"
+  sha256 "09cd3cda9d3be66d307445fb4adefc2621460102e672edb8496b1c900237ab90"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "yarn" => :build
+
+  def install
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/replicatedhq/ship").install Dir["*"]
+
+    ENV["GOPATH"] = gopath
+    ENV["GOBIN"] = bin
+    ENV.prepend_path "PATH", bin
+
+    (gopath/"bin").mkpath
+
+    cd gopath/"src/github.com/replicatedhq/ship" do
+      ENV.deparallelize { system "make", "VERSION=#{version}", "GIT_SHA=\"\"", "build-minimal" }
+      bin.install "bin/ship"
+    end
+  end
+
+  test do
+    system "#{bin}/ship", "version"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
Additional tests added.